do.sh

```diff
@@ -165,6 +165,20 @@ e2e_build_library() {
   echo "Done."
 }
 
+e2e_build_library_ctx2() {
+  echo "Building End-To-End library with Context2 API into $BUILD_DIR/library..."
+  e2e_assert_dependencies
+  set -e
+  e2e_assert_jsdeps
+  BUILD_EXT_DIR="$BUILD_DIR/library"
+  rm -rf "$BUILD_EXT_DIR"
+  mkdir -p "$BUILD_EXT_DIR"
+  e2e_build_closure_lib_ "e2e.openpgp.Context2Impl" "$BUILD_EXT_DIR/end-to-end.compiled.js"
```

Comment:
Change the output filenames such that they don't overwrite the ctx1 ones.
We could then make a build target that builds both.